transit_odp/site_admin/tasks.py

```diff
@@ -168,3 +170,13 @@ def task_delete_unwanted_data():
         .delete()
     )
     logger.info(f"Deleted {deleted} post publishing check weekly report")
+
+
+@shared_task()
+def task_seasonal_service_updated_dates():
+    today = timezone.datetime.today()
+    one_year_deltatime = timedelta(days=365)
```

Comment:
It might be an idea to use relative delta and 1 year. I think that will take leap years into account.

Reply:
This is really a good idea, but I got this problem:

`ProgrammingError: can't adapt type 'relativedelta'`

`start` and `end` are `DateField` don't accept `relativedelta`.
This can be avoid if :
1. QuerySet
2. sum the dates
3. save the data in db

but this destroy the single QuerySet
@ITO-PaulHolland

Reply:
damn it. leave it like this. I think its ok